src/lib/Bdash.ts

```diff
@@ -1,13 +1,16 @@
 import fs from "fs";
-import Config from "./Config";
 import { ensureDirSync } from "fs-extra";
 import { setting } from "./Setting";
 import Database from "./Database";
+import { ipcRenderer } from "electron";
 
 const Bdash = {
   async initialize(): Promise<void> {
     // @see https://github.com/bdash-app/bdash/pull/99#issuecomment-590011101
     window.process["browser"] = true;
+
+    const Config = await ipcRenderer.invoke("getConfig");
```

Comment:
`Config` object uses `app` module, so I added `getConfig` channel and changed to make `Config` object in main process.